### DISCUSSION TITLE
preview _config.yml before commit?

### DISCUSSION THREAD

**Discussion by j-hmd** [community/community#21820] [Other Feature Feedback, Questions, & Ideas]
<p>Hi!</p>
<p>I’m new to github and github pages, so I was wondering if there is a way to preview my github page before commiting and pushing it.</p>
<p>I think I’m pushing too many commits for small things like “added a space” or “tried this syntax” in my github pages. </p>
<p>Appreciate any help!</p>

**Comment by AndreaGriffiths11** [comment, marked as answer]
<p>Hi <a class="mention" href="/j-hmd">@j-hmd</a>,</p>
<p>Thanks for being here and our apologies for the delay in response. You can build your GitHub Pages site locally to preview and test changes to your site. Here is a link to the instructions: <a href="https://help.github.com/en/github/working-with-github-pages/testing-your-github-pages-site-locally-with-jekyll#building-your-site-locally" rel="nofollow noopener">https://help.github.com/en/github/working-with-github-pages/testing-your-github-pages-site-locally-with-jekyll#building-your-site-locally</a></p>

**Comment by j-hmd** [comment]
<p>Thank you <a class="mention" href="/andreagriffiths11">@andreagriffiths11</a> !</p>
<p>I followed the steps to render jekyll in my local machine! Works (:</p>
<ul>
<li>J</li>
</ul>
